app/models/sponsor.rb

```diff
@@ -30,7 +30,9 @@ class Sponsor < ActiveRecord::Base
   validate :date_not_beyond_first_of_next_month
   validate :belongs_to_one_branch_or_organization
   validate :can_be_inactivated, if: :being_inactivated?, on: :update
-  validates_format_of :email, with: Devise.email_regexp, allow_blank: true
+  validates_format_of :email,
+            with: /\A[_a-z0-9](\.?[\-+_%a-z0-9]+)*@([_+\-%a-z0-9]+)(\.[_+\-%a-z0-9]+)*(\.[_a-z0-9]+)+\z/i,
+            allow_blank: true
   validate :type_matches_affiliation, on: :create
```

Comment:
Devise: `email_regexp` should be used to give user feedback and not to assert the e-mail validity.
See here:
https://github.com/plataformatec/devise/blob/c19f1f27cecd7024df0ff8b4ace400d665dd9576/lib/devise.rb#LC119